p4-fusion/labels_conversion.h

```diff
@@ -0,0 +1,12 @@
+#ifndef LABELS_CONVERSION_H
+#define LABELS_CONVERSION_H
+#include "p4_api.h"
+
+int updateTags(P4API* p4, const std::string& depotPath, git_repository* repo);
+std::string convertLabelToTag(std::string label);
+std::string trimSuffix(const std::string& str, const std::string& suffix);
+std::string trimPrefix(const std::string& str, const std::string& prefix);
+std::string getChangelistFromCommit(const git_commit* commit);

```

Comment:
do you need to define those in the header if they're only used in the implementation (C++ noob, I forgot if that is true or not 😬)